### PR TITLE
2822 oxidation replace addrs

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
depends on #2815

### CODE REVIEW

File: parsec/api/protocol/invite.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 from enum import Enum
-from typing import Dict, cast, TYPE_CHECKING
+from typing import Dict, cast
 
-from parsec.types import UUID4
 from parsec.serde import BaseSchema, OneOfSchema, fields
 from parsec.api.protocol.base import BaseReqSchema, BaseRepSchema, CmdSerializer
 from parsec.api.protocol.types import HumanHandleField, UserIDField
 
+from parsec._parsec import (
```
Comment (TimeEngineer) [inline parsec/api/protocol/invite.py:10]: Why parenthesis ?

File: parsec/_parsec/addrs.pyi [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    def __init__(
+        self,
+        organization_id: OrganizationID,
+        hostname: str,
+        port: Optional[int],
+        use_ssl: bool = True,
+    ) -> None: ...
+    def __repr__(self) -> str: ...
+    def __str__(self) -> str: ...
+    def __eq__(self, other: BackendInvitationAddr) -> bool: ...
```
Comment (TimeEngineer) [inline parsec/_parsec/addrs.pyi:229]: Can't we compare with BackendPkiEnrollmentAddr ? Is the binding missing too ?

**Review by TimeEngineer — CHANGES_REQUESTED** [review]

File: parsec/_parsec/addrs.pyi (hunk 2 above)
Comment (FirelightFlagboy) [reply parsec/_parsec/addrs.pyi:229]: That's a typo

File: parsec/api/protocol/invite.py (hunk 1 above)
Comment (FirelightFlagboy) [reply parsec/api/protocol/invite.py:10]: Missing `black` formatting

File: parsec/_parsec/addrs.pyi (hunk 2 above)
Comment (FirelightFlagboy) [reply parsec/_parsec/addrs.pyi:229]: done

File: parsec/api/protocol/invite.py (hunk 1 above)
Comment (FirelightFlagboy) [reply parsec/api/protocol/invite.py]: done, but I've to manually format this line

**Review by TimeEngineer — APPROVED** [review]

**Pull request merged**